packages/lsp8-contracts/hardhat.config.ts

```diff
@@ -39,6 +43,27 @@ function getTestnetChainConfig(): NetworkUserConfig {
   return config;
 }
 
+function getMainnetChainConfig(): NetworkUserConfig {
+  const config: NetworkUserConfig = {
+    live: true,
+    url: 'https://42.rpc.thirdweb.com',
```

Comment:
Let's change it to

```suggestion
    url: 'https://rpc.mainnet.lukso.network',
```

(including in the other files